install-php-extensions

```diff
@@ -1304,7 +1304,11 @@ installPECLModule() {
 	case "$2" in
 		http)
 			# http must be loaded after raphf and propro
-			docker-php-ext-enable --ini-name "zzz-$2.ini" "$2"
+			docker-php-ext-enable --ini-name "xx-php-ext-$2.ini" "$2"
```

Comment:
Why did you replace `zzz` with `xx-php-ext`?

Reply:
I replaced `zzz` to` xx-php-ext` so that the extensions loading before zz-docker.ini.

If you want, I will rollback this change (for http and memcached).

Reply:
No, that's fine. Thanks for this fix